src/Core/src/Core/ICheckBox.cs

```diff
@@ -1,7 +1,14 @@
 ﻿namespace Microsoft.Maui
 {
+	/// <summary>
+	/// Represents a View that is a type of button that can either be
```

Comment:
A checkbox is not a button. 

Perhaps "Represents a View which allows the user to select a binary choice"?

Reply:
Yeah, your definitions conveys better what it is.